convergence-arrow/Cargo.toml

```diff
@@ -12,7 +12,7 @@ tokio = { version = "1" }
 async-trait = "0.1"
 datafusion = "43"
 convergence = { path = "../convergence", version = "0.16.0" }
-chrono = "0.4"
+chrono = "=0.4.39"
```

Comment:
Why the pin here?

Reply:
without this i got:
`error[E0034]: multiple applicable items in scope` 
for `ChronoDateExt::quarter | Datelike::quarter` 
but now it works fine

reverted